plugin.xml

```diff
@@ -10,22 +10,59 @@
     <keywords>cordova,urbanairship</keywords>
     <repo>https://github.com/urbanairship/urbanairship-accengage-cordova.git</repo>
     
+    <engines>
+        <engine name="cordova-android" version=">=4.1.0"/>
+        <engine name="cordova-ios" version=">=5.0.1"/>
+        <engine name="cordova" version=">=9.0.1"/>
+        <!-- <engine name="urbanairship-cordova" version=">=13.1.0"/> -->
```

Comment:
I commented this line for the moment until figuring out why I have this error when I try to add the `urbanairship-cordova` dependency.

`A-C02VP0GQG8WL:test mkhemiri$ cordova plugin add ../../urbanairship-accengage-cordova/
Installing "urbanairship-accengage-cordova" for android
warn`

Reply:
Does this work now with the correct version?

Reply:
Still not working even with the correct version :/

Reply:
Version 10.1 has been released. Hopefully this will fix your issue.

Reply:
Finally it works 👍  but it's not related to the new version. The problem is in the engine tag, it misses some required attributes for custom frameworks (platform and scriptSrc). The format of custom framework should be like this : 
`<engines>`
`<engine name="my_custom_framework" version="1.0.0" platform="ios|android" scriptSrc="path_to_my_custom_framework_version"/>`
 `</engines>`